examples/features/annotated-namespace/README.md

```diff
@@ -0,0 +1,144 @@
+# Test kernel to kernel connection with annotated namespace
+
+This example shows that NSM annotations applied to namespace will be applied to the pods within this namespace.  
+
+NSC and NSE are using the `kernel` mechanism to connect to its local forwarder.
+
+
+## Requires
+
+Make sure that you have completed steps from [basic](../../basic) or [memory](../../memory) setup.
+
+## Run
+
+Create test namespace without annotation:
+```bash
+kubectl create ns ns-annotated-namespace
+```
+
+Select node to deploy clients:
+```bash
+NODE=($(kubectl get nodes -o go-template='{{range .items}}{{ if not .spec.taints  }}{{index .metadata.labels "kubernetes.io/hostname"}} {{end}}{{end}}')[0])
```

Comment:
Do we need a node selection for this example?

Reply:
I checked it locally and it worked without node selection